tests/instruments/test_ship_underwater_st.py

```diff
@@ -80,21 +81,23 @@ def test_simulate_ship_underwater_st(tmpdir: py.path.LocalPath) -> None:
         sample_points=sample_points,
     )
 
+    # test if output is as expected
     results = xr.open_zarr(out_path)
 
-    # below we assert if output makes sense
-    assert len(results.trajectory) == 1  # expect a singe trajectory
+    assert len(results.trajectory) == 1  # expect a single trajectory
     traj = results.trajectory.item()
     assert len(results.sel(trajectory=traj).obs) == len(
         sample_points
     )  # expect as many obs as sample points
 
     # for every obs, check if the variables match the expected observations
-    for obs_i, exp in zip(results.sel(trajectory=traj).obs, expected_obs, strict=True):
+    for i, (obs_i, exp) in enumerate(
```

Comment:
explicitly iterate over i, because printing obs_i in case of failed assert turned out to not be useful, and with i we can tell which observation failed.